sdk/personalizer/ai-personalizer-rest/package.json

```diff
@@ -30,7 +30,8 @@
     "types/ai-personalizer.d.ts",
     "README.md",
     "LICENSE",
-    "review/*"
+    "review/*",
+    "CHANGELOG.md"
```

Comment:
analyze check failed without this because we need to publish the changelog.

Reply:
still fails.., still looking

Reply:
turned out the ci.yml has bad artifacts names, fixed that.

thanks @ckairen

Reply:
Why is CHANGELOG.md needed now? Should we remove review/*?

Reply:
We do ship changelog files, right?